contrib/cron-jobs/qualle.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# How often should this run?
+#  Once a day max
+
+# crontab entry for this (proposal):
+# # m h  dom mon dow   command
+# 02  01 *   *   *     ( cd /tmp; /my/path/to/this/script )
+#
+
+cd /tmp
+mkdir qualle
+cd qualle
+
+echo "Download IPv4 table from 'qualle.cert.at'."
+whois -h qualle.cert.at block origin v4table > ipasn.dat
+echo "Download IPv6 table from 'qualle.cert.at'."
+whois -h qualle.cert.at block origin v6table >> ipasn.dat
+
+echo "Convert format"
+sed -i 's/^v[46]*table //' ipasn.dat
+sed -i 's/ /\t/' ipasn.dat
+
+echo "Perform sanity checks"
+if [[ $(wc -l ipasn.dat | awk '{print $1}') -lt 650000 ]]; then
+    echo "Less then 650k entries, can't be true."
+    exit 1
+fi
+
+python3 -c "import pyasn; pyasn.pyasn('/tmp/qualle/ipasn.dat')" > /dev/null
+if [[ $? -ne 0 ]]; then
+    echo "Unparseable data."
+    exit 1
+fi
+
+echo "Applying new data to '/opt/intelmq/var/lib/bots/asn_lookup/'."
+mv ipasn.dat  /opt/intelmq/var/lib/bots/asn_lookup/
```

Comment:
Any other sanity checks needed before moving it there?

Does the bot need to be HUPed?